INSTALL.md

```diff
@@ -34,91 +35,41 @@ apt-get update
 
 ### Installation
 ```bash
-wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-node_linux_amd64.deb
-sudo dpkg --install --force-depends mysterium-node_linux_amd64.deb
+wget https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/myst_linux_amd64.deb
+sudo dpkg --install --force-depends myst_linux_amd64.deb
 sudo apt-get install -y --fix-broken
 ```
+
 ### Running service
 ```bash
 sudo service mysterium-node start
 sudo service mysterium-node status
 ```
+
 ### Debugging service
 ```bash
 sudo tail -f /var/log/mysterium-node/*
 ```
-### Debugging standalone
-```bash
-sudo mysterium_server --data-dir=/var/lib/mysterium-node --config-dir=/etc/mysterium-node --runtime-dir=/tmp --identity=0x123456..
-```
-
-
-## Mysterium VPN client (Debian && Ubuntu) - tested on Ubuntu 16.04
-### Download
- * https://github.com/mysteriumnetwork/node/releases/download/{VERSION}/mysterium-client_linux_amd64.deb
-
-### Add latest OpenVPN repository
-
-```bash
-apt-get update && apt-get install -y curl
-curl -s https://swupdate.openvpn.net/repos/repo-public.gpg | apt-key add && echo "deb http://build.openvpn.net/debian/openvpn/stable xenial main" > /etc/apt/sources.list.d/openvpn-aptrepo.list && rm -rf /var/cache/apt/* /var/lib/apt/lists/*
```

Comment:
Why to remove these instructions?

Reply:
Removed section for client download and installation. Since there are no more dedicated client binary and installation process, only instruction for installing single binary left.